frontend/src/pages/Inspection/Comments.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+import { Comment } from './Comment';
+
+const Wrapper = styled.div`
+  padding-left: 0.5em;
+  border-left: 0.1em solid #e4e4e4;
+  max-width: 30em;
+  min-width: 15em;
+`;
+
+export const Comments = (props) => {
+  const { comments } = props;
+  return (
+    <Wrapper>
+      <h2>Feedback</h2>
+      {comments.map(({
+        id, comment, author, date,
+      }) => (
+        <Comment
+          key={id}
+          comment={comment}
+          author={author}
+          date={date}
+        />
+      ))}
+    </Wrapper>
+  );
+};
+
+Comments.propTypes = {
+  comments: PropTypes.arrayOf(PropTypes.object).isRequired,
```

Comment:
Do you know how what the structure of the object will be?  If so then it could be better with PropTypes.shape({...})